custom/icds_reports/tasks.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+from django.conf import settings
+
+from django.db import connections
+
+
+@periodic_task(run_every=crontab(minute=0, hour=0, day_of_week=7), acks_late=True)
+def move_ucr_data_into_aggregation_tables():
+    with connections[settings.ICDS_UCR_DATABASE_ALIAS].cursor() as cursor:
```

Comment:
I believe this task will only run in environments where this app (icds_reports) is present in INSTALLED_APPS, and that should only be the case for the india server (where `ICDS_UCR_DATABASE_ALIAS` should also exist).

So, this could 500 if the server ever gets deployed wrong, which I think would be preferable to a more silent failure. I'll add an assert though with a custom message to make this clear.